doc/source/whatsnew/v0.25.0.rst

```diff
@@ -387,8 +387,7 @@ MultiIndex
 ^^^^^^^^^^
 
 - Bug in which incorrect exception raised by :class:`Timedelta` when testing the membership of :class:`MultiIndex` (:issue:`24570`)
--
--
+- Bug in :class:`Multindex` construction from levels and codes that would incorrectly allows code values < -1 or NaN levels (:issue:`19387`)
```

Comment:
May need to move this under the "API breaking changes" section, as previously we did allow codes < 0?.

Reply:
yeah sure

Reply:
Moved the whatsnew entry to an API breaking change